### PR TITLE
convert_angle

### PR DESCRIPTION
**Pull request by inagen**
convert_angle func

### CODE REVIEW

File: src/commands/convert.cpp [hunk 1]
```diff
@@ -36,6 +36,22 @@ double convert_temp(std::string in_what, std::string data){
     throw std::exception();
 }
 
+double convert_angle(std::string in_what, std::string data){
```
Comment (foxcpp) [inline src/commands/convert.cpp]: Passing `std::string` by value causes unnecessary copy. Prefer to pass by const reference (`const std::string&`) if you need only to observe value.

Comment (foxcpp) [inline src/commands/convert.cpp]: Code style violation: There is missing space after closing brace.

Comment (foxcpp) [inline src/commands/convert.cpp]: `in_what` parameter is not obvious at all. Why not to define separate functions for each conversion and perform comparsion in `void convert(const TgBot::Message::Ptr& message)`?

**Comment by HexwellC** [comment]
Please, stop making pull request into master for no reason.

**Pull request closed without merging**